### PR TITLE
Add a fetch-jobs.rb script to Jenkins tools

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

**Comment by ancorgs** [comment]
LGTM

**Comment by imobach** [comment]
Thanks.

File: jenkins-tools/fetch-jobs.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#
+#   ruby fetch-jobs.rb --config ci-jenkins.yml --prefix yast- --output configs
+#
+# You can get more information using the --help option:
+#
+#   ruby fetch-jobs.rb --help
+begin
+  require "jenkins_api_client"
+rescue LoadError
+  STDERR.puts "You need to install rubygem-jenkins_api_client package"
```
Comment (lslezak) [inline jenkins-tools/fetch-jobs.rb]: NP: You should prefer `$stderr` instead, `STDERR` is a constant and cannot be redefined while `$stderr` is a global variable which can be modified at runtime (you can easily redirect the output or you can capture the output and verify it in tests...).

**Comment by lslezak** [comment]
BTW is this intended for SP1?

**Pull request merged**